epoxy-processor/src/main/java/com/airbnb/epoxy/AttributeInfo.java

```diff
@@ -231,7 +233,57 @@ public int hashCode() {
   }
 
   boolean isViewClickListener() {
-    return isViewClickListenerType(getTypeMirror()) || isViewLongClickListenerType(getTypeMirror());
+    return isViewClickListenerType(getTypeMirror());
+  }
+
+  boolean isViewLongClickListener() {
+    return isViewLongClickListenerType(getTypeMirror());
+  }
+
+  boolean isBoolean() {
+    return getTypeMirror().getKind() == TypeKind.BOOLEAN;
+  }
+
+  boolean isCharSequenceOrString() {
+    return Utils.isCharSequenceOrStringType(getTypeMirror());
+  }
+
+  boolean isStringList() {
+    return Utils.isType(getTypeMirror(), "java.util.List<java.lang.String>")
```

Comment:
maybe a good way to DRY up the two list checks is to have a `isList(type)` method that checks for java/kotlin lists and also checks the generic type as a parameter